module/VuFind/src/VuFind/Controller/HelpController.php

```diff
@@ -52,9 +52,14 @@ class HelpController extends AbstractBase
      */
     public function homeAction()
     {
+        $topic = $this->params()->fromRoute('topic');
+        if (empty($topic)) {
```

Comment:
It might be a good idea to add a special case for `$topic === 'Home'` -- right now, things are working because the help-home route gets matched for /Help/Home, but if we eventually remove that legacy route but somebody continues linking to a legacy URL, I think it will end up coming here and resolving to a topic of 'Home' which will cause problems. I don't think it hurts to cover that scenario, just in case!

Reply:
I added a minor change, however i'm not sure whether that covers the case that you meant correctly. Please have another look.

Reply:
What I had in mind was more like:

```
if (empty($topic) || $topic === 'Home') {
    $topic = $this->params()->fromQuery('topic');
}
```

If the route topic resolves to Home, then the real topic is in the query and we need to retrieve it.

You can test this scenario by accessing an old-style URL after removing the 'Help/Home' static route from the configuration.

Reply:
Understood, thanks! The code has been adjusted accordingly.